src/components/OtherPlayerInfoComponent.js

```diff
@@ -16,14 +16,17 @@ function OtherPlayerInfo(props) {
 							<Container style={{ width: "20em" }}>
 								<Row>
 									<Col xs={6}>
-										{(props.data.cards.length >= 1) ? <PlayCard cardName={"back"}/> : <div/>}
+										{(props.data.cards.length >= 1) ? <PlayCard cardName={"back"}/> : <div>Out Of game, what a loser!</div>}
 									</Col>
 									<Col xs={6}>
 										{(props.data.cards.length == 2) ? <PlayCard cardName={"back"}/> : <div/>}
 									</Col>
 								</Row>
 							</Container>
 						</div>
+						<div align = "center" style = {{marginTop:10}}>
```

Comment:
use `em` and not just `10`. By default i think it uses `px` which can change by screen.